package.json

```diff
@@ -53,23 +53,23 @@
     "@babel/preset-env": "^7.6.2",
     "@babel/preset-react": "^7.0.0",
     "babel-eslint": "^10.0.3",
-    "babel-jest": "^24.9.0",
+    "babel-jest": "^25.4.0",
     "babel-preset-env": "^1.7.0",
     "babel-preset-react": "^6.23.0",
     "enzyme": "^3.10.0",
     "enzyme-adapter-react-16": "^1.14.0",
     "eslint": "^6.5.1",
     "eslint-config-airbnb": "^18.0.1",
-    "eslint-config-prettier": "^6.3.0",
-    "eslint-plugin-import": "^2.18.2",
+    "eslint-config-prettier": "^6.10.1",
+    "eslint-plugin-import": "^2.20.2",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.15.0",
-    "jest": "^24.9.0",
-    "node-sass": "^4.12.0",
-    "prettier": "^1.18.2",
-    "react": "^16.10.1",
+    "jest": "^25.4.0",
+    "node-sass": "^4.13.1",
+    "prettier": "2.0.4",
```

Comment:
i think your linting might fail, its worth running `npm run lint:prettier` locally to see if its throwing any errors.